cmd/tailscale/cli/funnel.go

```diff
@@ -117,6 +130,33 @@ func (e *serveEnv) runFunnel(ctx context.Context, args []string) error {
 	return nil
 }
 
+func (e *serveEnv) streamFunnel(ctx context.Context, hp ipn.HostPort) error {
+	sc, err := e.lc.GetServeConfig(ctx)
+	if err != nil {
+		return err
+	}
+	if sc == nil {
+		sc = new(ipn.ServeConfig)
+	}
+	if sc.AllowFunnel[hp] {
+		return fmt.Errorf("Funnel already running on %q", hp)
+	}
```

Comment:
`if sc == nil`, then we can return early as there's no active `ServeConfig`

Reply:
I think it's ok to treat this the same as if we use `stream` but don't have a serve config for the hostport (even if `sc != nil`). In that case we show the warning message, but still start things up:
```
Warning: funnel=on for tardos-2.sando.ts.net:443, but no serve config
         run: `tailscale serve --help` to see how to configure handlers
Funnel started on "https://tardos-2.sando.ts.net:443".
Press Ctrl-C to stop Funnel.
```
(same thing that happens when you are just turning funnel on for a port that lacks config - warns but doesn't fail)